platforms/r3-corda/configuration/roles/setup/notary/tasks/main.yaml

```diff
@@ -102,6 +102,7 @@
     GIT_DIR: "{{ playbook_dir }}/../../../"
     GIT_REPO: "{{ gitops.git_push_url }}"
     GIT_USERNAME: "{{ gitops.username }}"
+    GIT_EMAIL: "{{ gitops.email }}"
```

Comment:
the "email" key needs to be added in the sample network.yaml files in platforms/<platform>/configuration/samples directory. Also docs/source updated with this info.

Reply:
Oh Thanks for pointing out ! Now I have done the changes for the existing 3 platforms